### ISSUE TITLE
Generic styles affecting other website elements

### ISSUE THREAD

**Issue by bbceg** [Blink-Payment/Blink-WooCommerce-v2#3]
Hello,

The stylesheet for this plugin targets all links, buttons and inputs which has a broader effect than is perhaps intended.

The code in question can be seen here:

![image](https://github.com/user-attachments/assets/29d7bec0-c8e3-439f-b13a-fe6a6d4122b4)

![image](https://github.com/user-attachments/assets/1c038765-59c0-4ed4-a2a6-86cf122a3a8c)

Would it be possible to make these plugin specific?

Thanks

**Comment by shalomMoiseBlink** [comment]
Thanks for raising @bbceg . We will look into changing the styling to use only specific element classes. We'll update on this thread when it is live.

**Comment by shalomMoiseBlink** [comment]
@bbceg can you please provide the file-path that you are getting this css from?

**Comment by bbceg** [comment]
My apologies, it seems I am on version 1.0.11. 🤦‍♂️ I will update. 

Any chance this plugin could be added to the official WordPress repositories to streamline updates / reduce the chance of missing that an update is available (as I did here!).

**Comment by shalomMoiseBlink** [comment]
Hi @bbceg ,

Thanks for letting us know. We are working on publishing on Wordpress directly. In the meantime, to get notified on a new release, you can watch this repository.

**Comment by bbceg** [comment]
Good point, thanks @shalomMoiseBlink not sure why I didn't think of that!
